src/test/java/com/wikia/webdriver/common/core/api/GraphApi.java

```diff
@@ -25,89 +28,84 @@
 
 public class GraphApi {
 
-    private static final String ERROR_MESSAGE = "Problem with Graph API call used to create new facebook test user";
-    private static final String URI_SYNTAX_EXCEPTION = "URI_SYNTAX EXCEPTION";
-    private static ArrayList<BasicNameValuePair> PARAMS;
-    private static final String wikia_production_app_access_token = XMLReader
-        .getValue("ci.user.facebook.prod.accessToken");
-
-    public HashMap<String, String> createFacebookTestUser(String appId) {
-        try {
-            HttpResponse response = createTestUser(appId);
-            String entity = EntityUtils.toString(response.getEntity());
-            return new Gson().fromJson(entity, new TypeToken<HashMap<String, String>>(){}.getType());
-        } catch (IOException e) {
-            PageObjectLogging.log(URI_SYNTAX_EXCEPTION, ExceptionUtils.getStackTrace(e), false);
-            throw new WebDriverException(ERROR_MESSAGE);
-        } catch (URISyntaxException e) {
-            PageObjectLogging.log(URI_SYNTAX_EXCEPTION, ExceptionUtils.getStackTrace(e), false);
-            throw new WebDriverException(ERROR_MESSAGE);
-        }
-
+  private static final String ERROR_MESSAGE = "Problem with Graph API call used to create new facebook test user";
+  private static final String URI_SYNTAX_EXCEPTION = "URI_SYNTAX EXCEPTION";
+  private static final String wikiaProductionAppAccessToken = XMLReader
+    .getValue("ci.user.facebook.prod.accessToken");
+  private static final String wikiaProductionAppId = XMLReader
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)